reco_utils/docker/pyspark/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
are you planning to do 3 PRs: one for each env? or are you planning to use this one?